contributing/code/maintenance.rst

```diff
@@ -0,0 +1,58 @@
+Maintenance
+===========
+
+During the lifetime of a minor version, new releases (patch versions) are
+published on a monthly basis. This document describes the boundaries of
+acceptable changes.
+
+**Bug fixes** are accepted under the following conditions:
```

Comment:
This may be unclear for a *newbie*. When someone add a PR, what makes it a bug fix? Does **Bug fixes** have a special meaning in the context of Symfony? Is it mandatory that a bug fix refer to an existing ticket?